manifests/plugin/ansible/runner.pp

```diff
@@ -0,0 +1,48 @@
+# = Ansible runner package installation
+#
+# $manage_runner_repo:: If true, adds upstream repositories to install ansible-runner package from
+#
+class foreman_proxy::plugin::ansible::runner(
+  Boolean $manage_runner_repo = $foreman_proxy::plugin::ansible::manage_runner_repo,
+) {
+
+  if $manage_runner_repo {
+    case $::osfamily {
+      'Debian': {
+        include ::apt
+        ::apt::source { 'ansible-runner':
+          repos    => 'main',
+          location => 'https://releases.ansible.com/ansible-runner/deb',
+          key      => {
+            id     => 'AC48AC71DA695CA15F2D39C4B84E339C442667A9',
+            source => 'https://releases.ansible.com/keys/RPM-GPG-KEY-ansible-release.pub',
+          },
+          include  => {
+            src => false,
+          },
+        }
+        $package = 'ansible-runner'
+      }
+      'RedHat': {
+        yumrepo { 'ansible-runner':
+          descr    => 'Ansible runner',
+          baseurl  => "https://releases.ansible.com/ansible-runner/rpm/epel-${facts['os']['release']['major']}-${facts['architecture']}/",
+          gpgcheck => true,
+          gpgkey   => 'https://releases.ansible.com/keys/RPM-GPG-KEY-ansible-release.pub',
+          enabled  => '1',
+        }
+        $package = 'ansible-runner'
+      }
+      default: {
+        $package = 'python-ansible-runner'
```

Comment:
Seeing as how all of them are `ansible-runner`, is this for a BSD? If you don't know, I'd start with simply adding a class parameter `String $package_name = 'ansible-runner'` to keep it simple.

Reply:
It was just some dark thoughts leading me that direction, I added warning for other OSs, you think it is enough?